blockchain/src/normal.rs

```diff
@@ -271,13 +272,19 @@ impl cbor::CborValue for VssCertificates {
             .embed("while decoding VssCertificates")
     }
 }
+impl VssCertificates {
+    pub fn iter(&self) -> ::std::slice::Iter<VssCertificate> {
+        self.0.iter()
+    }
+}
+
 
 #[derive(Debug, Clone)]
 pub struct VssCertificate {
-    vss_key: cbor::Value, // TODO more than just cbor
-    expiry_epoch: types::EpochId,
-    signature: cbor::Value,   // TODO more than just cbor
-    signing_key: cbor::Value, // TODO more than just cbor
+    pub vss_key: cbor::Value, // 35 bytes public-key from http://hackage.haskell.org/package/pvss/docs/Crypto-SCRAPE.html#t:Point
+    pub expiry_epoch: types::EpochId,
+    pub signature: redeem::Signature,
```

Comment:
The size might be the same, but the meaning is completely different.
In wallet_crypto creates a new module named `vss.rs` and add the missing placeholder (no need to add the crypto yet).